src/docs/styles/pages/_home.scss

```diff
@@ -7,7 +7,7 @@
 
   &::after {
     bottom: calc(0rem - space(sm));
-    z-index: 1;
+    z-index: 10;
```

Comment:
can we stick with 1 on these since we likely would only want this to be positioned directly on top of the parent but not above anything else someone may put in between?